ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java

```diff
@@ -94,6 +97,13 @@ public MarkSweepPruner(
             BesuMetricCategory.PRUNER,
             "sweep_operations_total",
             "Total number of sweep operations performed");
+
+    markStopwatch = Stopwatch.createUnstarted();
+    metricsSystem.createLongGauge(
+        BesuMetricCategory.PRUNER,
+        "mark_time_duration",
+        "Total number of seconds spent marking the state trie",
```

Comment:
Cumulative or per marking run?  Worth adding some verbiage to the help text to make it clear.

Reply:
resolved